plugins/linux-pulseaudio/pulse-wrapper.c

```diff
@@ -185,6 +185,28 @@ int_fast32_t pulse_get_source_info_list(pa_source_info_cb_t cb, void* userdata)
 	return 0;
 }
 
+int_fast32_t pulse_get_sink_info_list(pa_sink_info_cb_t cb, void* userdata)
```

Comment:
The signature is copied from the existing function for sources, if i change it then the style in the file is inconsistent.